README.md

```diff
@@ -15,13 +12,14 @@ Traceback (most recent call last):
 ImportError: libnvjpeg2k.so.0: cannot open shared object file: No such file or directory
 ```
 
-you may need to copy the included `libnvjpeg2k.so` files to your system's library path. For example, run
+you may need to copy `libnvjpeg2k.so` files to your system's library path. For example, run
```

Comment:
We shouldn't need any references to libnvjpeg2k anymore. Everything should be handled automatically by installing nvidia-nvjpeg2k-cu12. We may also be able to add pybind as a dependency so that it is handled automatically too, in which case we can remove it from the troubleshooting steps.

Reply:
When I initially tested, installing `nvidia-nvjpeg2k-cu12` through `pdm` wasn't sufficient. I also tried handling `pybind11` within the project, but `CMake` wasn't able to find it (though it seems like we should be able to solve this through a change to the build script).